src/amo/components/Footer/index.js

```diff
@@ -204,39 +204,50 @@ export class FooterBase extends React.Component<InternalProps> {
             <ul className="Footer-links">
               <li>
                 <a
-                  className="Footer-lockwise-link"
-                  href={`https://www.mozilla.org/firefox/lockwise/${makeQueryStringWithUTM(
+                  className="Footer-browsers-link"
+                  href={`https://www.mozilla.org/firefox/browsers/${makeQueryStringWithUTM(
                     {
                       utm_content: 'footer-link',
                       utm_campaign: null,
                     },
                   )}`}
                 >
-                  Lockwise
+                  Browsers
```

Comment:
We were already not translating it before, but... should `Browsers` be left untranslated ?

Reply:
I have asked about this on Slack.

Reply:
It seems that the answer is "yes, leave it untranslated".

Reply:
:shrug: